recipes/15/15_interval_joins.md

```diff
@@ -0,0 +1,61 @@
+# 15 Interval Joins
```

Comment:
"Temporal Join" is actually the better known term for this. Interval Join is pretty Flink-specific, I think.

Reply:
Uff, but then isn't it confusing with "actual" temporal joins?

Reply:
Yeah, probably makes sense to keep it in line.